internal/olm/operator/bundle/install.go

```diff
@@ -87,6 +109,71 @@ func (i *Install) setup(ctx context.Context) error {
 		return err
 	}
 
+	var declcfg *declarativeconfig.DeclarativeConfig
+
+	// get index image labels.
+	catalogLabels, err := registryutil.GetImageLabels(ctx, nil, i.IndexImageCatalogCreator.IndexImage, false)
+	if err != nil {
+		return fmt.Errorf("get index image labels: %v", err)
+	}
+
+	// set the field to true if FBC label is on the image or for a default index image.
+	if _, hasFBCLabel := catalogLabels[containertools.ConfigsLocationLabel]; hasFBCLabel || i.IndexImageCatalogCreator.IndexImage == registry.DefaultIndexImage {
+		i.IndexImageCatalogCreator.HasFBCLabel = true
+	}
+
+	// generate an fbc if an fbc specific label is found on the image or for a default index image.
+	if i.IndexImageCatalogCreator.HasFBCLabel {
+		// FBC variables
+		f := &FBCContext{
+			Package:     labels[registrybundle.PackageLabel],
+			ChannelName: strings.Split(labels[registrybundle.ChannelsLabel], ",")[0],
```

Comment:
Going forward, bundles won't have to have channel metadata included. I think this makes sense _if_ the length of `strings.Split(labels[registrybundle.ChannelsLabel], ",")` is greater than 0. But we should handle a bundle that doesn't define channels as well. 

Question is: what channel name should we fall back to? Auto-generated with a random suffix? Some hardcoded name? The name of the CSV?

Any thoughts or considerations for any of those?

Reply:
I think anything prefixed by the operator name is a good fallback, ex: memcached-operator-alpha, nginx-operator-stable, etc.

Reply:
Yeah, that's another option. 

For options that use a non-random name: Would there be any concern with channel name conflicts if you run `operator-sdk run bundle` multiple times against different bundles in the same package?

For options that use a random name: Would there be any concern with the `operator-sdk run bundle` + `operator-sdk run bundle-upgrade` case?

It does seem like a non-random name as a fallback should work though. After all, if the bundle _does_ have a list of channels, we have been using something from that list in the existing `operator-sdk run bundle` code for sqlite-based indexes.

Reply:
```
{
    "schema": "olm.channel",
    "name": "alpha",
    "package": "nginx-operator",
    "entries": [
        {
            "name": "nginx-operator.v0.0.1"
        }
    ]
}
```

1. Non-random names - Is running run bundle successively for two different versions of the same operator supposed to be a use case? Each run bundle invocation creates a catalog source. The second time we invoke run bundle, the catalog source object of that operator will already exist, and it'll throw an error. Correct me if I understood your question incorrectly. 

2. Random names - Every channel is uniquely identified by both the name of the channel and its corresponding package.
So even if the channel.name isn't unique, channel.package _will_ be, and so I don't think there's should be any conflict. 

I'm leaning towards non-random names with a meaningful prefix. Anything related to auto generating a random string makes me slightly concerned about randomly generating the same string, even if it's unlikely.

Reply:
Hi @joelanford, @rashmigottipati @VenkatRamaraju 

I do **not** understand at all why we allow someone to pass the index via the run bundle option _(what is the point? Why/how/where a pre-existent index content maters and is useful data for the purpose of this feature?_ ). 

Assuming that I am right here and we never use the data from index informed ( the only diff is that we add the bundle to index informed instead of creating a new one when it is informed). I'd say why the channel matters? Could we not create any random/static name for all cases since the goal is only to check if the bundle can "run" with OLM?

Reply:
Btw, could we not remove this option from the run bundle and simplify it

Reply:
If the index content matter then, should not the channel also matters? If yes, how use a random or static value could address the need? How author would configure the channel if we are ignoring the annotations?

Reply:
Added a check to see if the channel metadata exists on the bundle. If not, setting it a hardcoded value. 
Fixed in [commit](https://github.com/operator-framework/operator-sdk/pull/5671/commits/20f18a072d3a82622896f88897ebf27253ec5284).